Resources/Prototypes/Entities/Clothing/Back/smuggler.yml

```diff
@@ -0,0 +1,1009 @@
+        prob: 0.05
+        orGroup: Chems
+      - id: SpaceCash
+        prob: 0.05
+        orGroup: Chems #5x tiny
+
+- type: entity
+  parent: ClothingBackpackSatchelSmuggler
+  id: ClothingBackpackSatchelSmugglerTwo
+  noSpawn: true
+  components:
+  - type: Sprite
+    sprite: Clothing/Back/Satchels/smuggler.rsi
+    state: icon
+  - type: StorageFill
+    contents:
+
+      - id: SpaceCash100
+        prob: 0.90
+        orGroup: Tobacco
+      - id: CigPackSyndicate
+        prob: 0.015
+        orGroup: Tobacco
+      - id: CigCartonGreen
+        prob: 0.025
+        orGroup: Tobacco
+      - id: CigCartonRed
+        prob: 0.025
+        orGroup: Tobacco
+      - id: CigCartonGreen
+        prob: 0.025
+        orGroup: Tobacco
+      - id: CigCartonBlack
+        prob: 0.025
+        orGroup: Tobacco
+      - id: CigarCase
+        prob: 0.025
+        maxAmount: 2
+        orGroup: Tobacco
+      - id: CigarGoldCase
+        prob: 0.025
+        maxAmount: 2
+        orGroup: Tobacco
+      - id: SpaceCash
+        prob: 0.025
+        orGroup: Tobacco #2x 2x3(12 slots)
+
+      - id: SpaceCash100
+        prob: 0.90
+        orGroup: Generic4
+      - id: GlowstickBase
+        prob: 0.1
+        maxAmount: 5
+        orGroup: Generic4
+      - id: GlowstickRed
+        prob: 0.1
+        maxAmount: 5
+        orGroup: Generic4
+      - id: GlowstickPurple
+        prob: 0.1
+        maxAmount: 5
+        orGroup: Generic4
+      - id: GlowstickYellow
+        prob: 0.1
+        maxAmount: 5
+        orGroup: Generic4
+      - id: GlowstickBlue
+        prob: 0.1
+        maxAmount: 5
+        orGroup: Generic4
+      - id: SpaceCash
+        prob: 0.1
+        orGroup: Generic4  #5x small
+
+      - id: SpaceCash100
+        prob: 0.85
+        orGroup: Clothing2
+      - id: ClothingEyesGlassesOutlawGlasses
+        prob: 0.15
+        orGroup: Clothing2
+      - id: ClothingEyesChameleon
+        prob: 0.15
+        orGroup: Clothing2
+      - id: ClothingEyesEyepatch
+        prob: 0.15
+        orGroup: Clothing2
+      - id: ClothingHandsGlovesNitrile
+        prob: 0.15
+        orGroup: Clothing2
+      - id: SpaceCash
+        prob: 0.15
+        orGroup: Clothing2  #1x small
+
+      - id: SpaceCash100
+        prob: 0.98
+        orGroup: GizmosPayload
+      - id: FlashPayload
+        prob: 0.02
+        maxAmount: 2
+        orGroup: GizmosPayload
+      - id: ChemicalPayload
+        prob: 0.02
+        maxAmount: 2
+        orGroup: GizmosPayload
+      - id: ExplosivePayload
+        prob: 0.02
+        maxAmount: 2
+        orGroup: GizmosPayload
+      - id: SpaceCash
+        prob: 0.02
+        orGroup: GizmosPayload #2x small
+
+      - id: SpaceCash100
+        prob: 0.98
+        orGroup: Circuitboards
+      - id: ChemDispenserMachineCircuitboard
+        prob: 0.05
+        orGroup: Circuitboards
+      - id: SyndicateMicrowaveMachineCircuitboard
+        prob: 0.05
+        orGroup: Circuitboards
```

Comment:
I'm pretty sure this is just a less useful microwave than the normal one. It might be slightly faster? I know it cant provide access.